Helper/Order.php

```diff
@@ -2483,7 +2483,13 @@ public function createOrderInvoice($order, $amount, $notify = false, $transactio
         $invoice->save();
 
         $order->addRelatedObject($invoice);
-        $order->setTotalPaid((float)$order->getTotalInvoiced() + $amount);
+
+        // in unexpected case order total paid could be empty, we should set it
+        if (!$order->getTotalPaid() && $order->getTotalInvoiced()) {
```

Comment:
Combine those two if together for readability
if (!$order->getTotalPaid()) {
 ...
 }